conda_forge_tick/migrators/core.py

```diff
@@ -68,7 +68,7 @@ def _get_source_code(recipe_dir):
     # Use conda build to do all the downloading/extracting bits
     md = render(recipe_dir, config=Config(**CB_CONFIG))
     if not md:
-        return
+        return None
```

Comment:
I'm curious how we're getting no metadata, should we write this into `bad`?

Reply:
it is a mini-migrator, so I figure we let other ones try at it.